### PR TITLE
dns/ddclient: Add Dyndns widget for new dashboard

### PR DESCRIPTION
**Pull request by Monviech**
Fixes: https://github.com/opnsense/plugins/issues/4142

### CODE REVIEW

File: dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            let colorClass = account.enabled === "1" ? 'text-success' : 'text-danger';
+            let tooltipText = account.enabled === "1" ? this.translations.enabled : this.translations.disabled;
+
+            let domainNames = account.hostnames.split(',')
+                  .map(domain => `<div>${domain}</div>`)
+                  .join('');
+
+            // Convert time to a localized format
+            let localizedTime = account.current_mtime ? new Date(account.current_mtime).toLocaleString() : '-';
+            let currentIp = account.current_ip || '-';
```
Comment (fichtner) [inline dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js:101]: For style reasons I prefer "N/A" whenever some value is expected but **n**ot **a**vailable, which also needs to be translated.

File: dns/ddclient/src/opnsense/www/js/widgets/Metadata/Dyndns.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<metadata>
+    <dyndnswidget>
```
Comment (fichtner) [inline dns/ddclient/src/opnsense/www/js/widgets/Metadata/Dyndns.xml:2]: "widget" is clear, and better to be precise that the origin is `ddclient`

File: dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js (hunk 1 above)
Comment (Monviech) [reply dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js:101]: https://github.com/opnsense/plugins/blob/cab23a684b902e5af00815a06bba7841a62d80a9/security/etpro-telemetry/src/opnsense/www/js/widgets/ETProTelemetry.js#L52

Comment (fichtner) [reply dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js:101]: let's make it a thing ;)

https://github.com/opnsense/plugins/blob/cab23a684b902e5af00815a06bba7841a62d80a9/dns/rfc2136/src/www/widgets/widgets/rfc2136.widget.php#L72

Comment (Monviech) [reply dns/ddclient/src/opnsense/www/js/widgets/Dyndns.js:101]: Sure np, I can fix it in the telemetry widget too, later. Gonna create an issue later.

**Review by fichtner — APPROVED** [review]
LGTM :)

**Comment by AdSchellevis** [comment]
@Monviech nice work, thanks!

**Pull request merged**